### DISCUSSION TITLE
Mobile -> Webview ?

### DISCUSSION THREAD

**Discussion by Mduroi** [DioxusLabs/dioxus#126] [Q&A] [answered]
First of all congratulations for the beginning of this project, I see a very good alternative to REACT in the near future.

My question is if the rendering on mobile will be done in webview too ? What approach?

**Comment by mrxiaozhuox** [comment]
Currently `dioxus-mobile` is immature, and its just `re-export` the `dioxus-desktop`, so it is same with the `desktop`, and its just support IOS system nowaday.

**Reply by mrxiaozhuox** [reply]
https://github.com/DioxusLabs/dioxus/blob/master/packages/mobile/src/lib.rs

**Comment by jkelleyrtp** [comment, marked as answer]
To answer the question - mobile rendering is currently being done with webview.

We are working on a fully native renderer, though the timeline for that is out in the distant future.
